dotCMS/src/main/java/com/dotcms/util/DotAssert.java

```diff
@@ -0,0 +1,393 @@
+package com.dotcms.util;
+
+import java.util.Collection;
+import java.util.Map;
+import java.util.function.Supplier;
+
+/**
+ * Handles assertions.
+ * @author jsanca
+ */
+public class DotAssert {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Add a private constructor to hide the implicit public one. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1118)